.gitignore

```diff
@@ -191,7 +191,7 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
+poetry.lock
```

Comment:
Would you split this part into another PR?
This is not related to linked issue.

Reply:
This is a correct suggestion! Since changing the poetry group name modifies the poetry.lock file, we should first merge the PR that adds poetry.lock to .gitignore.